### ISSUE TITLE
add via URL fails for large extensions

### ISSUE THREAD

**Issue by janLo** [coder/code-marketplace#66]
Large extensions like sonarlint (300+ MB) cannot be added via URL. I think the relevant part is this: https://github.com/coder/code-marketplace/blob/main/storage/storage.go#L358

I totally understand that there needs to be a limit and I see 100MB also as a good cutoff - but I do think, this should be mentioned in the docs.

**Comment by code-asher** [comment]
Agreed, and it sees reasonable to make a flag to change this as well.
